### DISCUSSION TITLE
Contributing to Anuvaad

### DISCUSSION THREAD

**Discussion by Asthasingh-2002** [project-anuvaad/anuvaad#918] [General] [answered]
hey @aravinth ,

I hope this message finds you well. I am writing to express my keen interest in contributing to the Indic Language Translation and Transliteration Library project. As a junior undergraduate in Information Technology specializing in Web Development, I am excited about the opportunity to make a meaningful contribution to this initiative.

Let me introduce myself. My name is Astha Singh and I possess a strong foundation in JavaScript. I have experience in developing user-friendly interfaces and documenting solution capabilities. Although I don't have prior experience in open-source projects specifically, I have actively participated in programming competitions and have contributed to open-source programs such as GirlScript Summer of Code.

To get started, I would appreciate your guidance on the next steps. Should I directly submit a proposal or is there a specific process to follow? I am eager to dive into the work, collaborate with the mentors, and make a meaningful contribution to the project.

Thank you for considering my interest in this project. I look forward to your response and the opportunity to collaborate with you and the project mentors.

Best Regards,
Astha Singh

_Originally posted by @Asthasingh-2002 in https://github.com/project-anuvaad/anuvaad/issues/547#issuecomment-1554344752_

**Comment by aravinth** [comment, marked as answer]
Hi @Asthasingh-2002 . Thanks for your interest and Apologies for delayed response. So far, project Anuvaad was focussed on a specific clients (Supreme/High Court of India, Supreme Court of Bangladesh and few others). But now, we are working on it to make it generic for all document translation usecases and we are welcoming open source contributions. 

Meanwhile, we are working on streamlining some of the process/documentation for the general public to contribute. 
Please let us know if you are still interested.

**Reply by Asthasingh-2002** [reply]
Thanks for getting back to me! 
I'm still very much interested in contributing. I appreciate the update about streamlining the process/documentation for contributors like myself. Please keep me posted on any developments or how I can get involved. Looking forward to being a part of this initiative.

**Comment by aravinth** [comment]
Hi @Asthasingh-2002 ,

We have created an issue to get started task : 
https://github.com/project-anuvaad/anuvaad/issues/919

The UI setup instructions are provided here : 
https://github.com/project-anuvaad/anuvaad/blob/master/anuvaad-fe/README.md

Let us know in case you are needing any help regarding this.

**Comment by Asthasingh-2002** [comment]
Hello Sir,

I hope this message finds you well.

I am encountering a few issues while attempting to clone the project to my
local environment. Could you please provide me with guidance on how to
properly set up the project? Thank you for your assistance.
Please find Screenshot of error attached below.
[image: cloneError.png]

Best regards,
Astha Singh

**Reply by aravinth** [reply]
Hi @Asthasingh-2002 - Unable to view the image.
Could you please re-attach the image?

**Comment by Asthasingh-2002** [comment]
Hello,
Sorry for inconvenience. Please find attached screenshot below:

Thanks and regards
Astha

**Reply by aravinth** [reply]
Still not visible. Can you please try pasting under https://github.com/project-anuvaad/anuvaad/issues/919 ?

**Comment by Asthasingh-2002** [comment]
I have paste under issue 919. Please check that.
